src/pkg/services/m365/api/lists.go

```diff
@@ -12,6 +13,8 @@ import (
 	"github.com/alcionai/corso/src/pkg/services/m365/api/graph"
 )
 
+var ErrInvalidTemplateError = fmt.Errorf("invalid list template(%s)", WebTemplateExtensionsListTemplateName)
```

Comment:
Per code standards, avoid formatting values into the error.  Also, this error is trying to be both generic ("invalid list template") and specific (webTemplateExtension) at the same time.  It should be one or the other.  I'd suggest the latter, in this case.
```suggestion
var ErrCannotCreateWebTemplateExtension = clues.New("unable to create webTemplateExtension type lists")
```